kivy/uix/textinput.py

```diff
@@ -1487,21 +1487,25 @@ def cancel_long_touch_event(self):
             self._long_touch_ev.cancel()
             self._long_touch_ev = None
 
+    def _select_word(self, delimiters=u' .,:;!?\'"<>()[]{}'):
+        ci = int(self.cursor_index())
+        cc = int(self.cursor_col)
```

Comment:
Not your fault, was already here, but there's a chance for a cleanup.
Do we feel comfortable about removing `int(..)` here?